### DISCUSSION TITLE
VIT model training set

### DISCUSSION THREAD

**Discussion by dmrangak** [mrdbourke/pytorch-deep-learning#619] [Q&A]
I hope to build a VIT model for scene recognition. I try to train it with GPU. I want to buy a computer, Please can you suggest a configuration for the computer to train the VIT model? (i have a funding facility )

**Comment by mrdbourke** [comment]
Hi @dmrangak , 

As for a computer to train a ViT model, I'd highly recommend something with a NVIDIA GPU.

For example, a NVIDIA 3080, 3090 or 4080/4090 would be great.

The more memory, the better.

For example, the NVIDIA 4080 has 16GB of memory where as the 4090 has 24GB of memory.

More memory = more capacity to train bigger models.

As for building a computer, I'd recommend https://pcpartpicker.com/ and see what kind of configurations you can start with.

I built my own deep learning PC a few years ago and you can read about it here: https://www.mrdbourke.com/notes-on-building-a-deep-learning-pc/
